### PR TITLE
Bump to version 1.0.0

### PR DESCRIPTION
**Pull request by abyss-w**
Bump to version 1.0.0

### CODE REVIEW

**Comment by abyss-w** [comment]
ping @Xuanwo, I don't have permission to add a tag and release version, could you please add a tag and release version?

**Comment by Xuanwo** [comment]
> ping @Xuanwo, I don't have permission to add a tag and release version, could you please add a tag and release version?

Yep, we will add it after this PR get merged.

**Comment by abyss-w** [comment]
> > ping @Xuanwo, I don't have permission to add a tag and release version, could you please add a tag and release version?
> 
> Yep, we will add it after this PR get merged.

OK!

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+The format is based on [Keep a Changelog](https://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](https://semver.org/).
+
+## v1.0.0 - 2021-09-08
+
+### Added
+
+- Implement bos services.
+
+[1.0.0]: https://github.com/beyondstorage/go-service-s3/compare/v1.0.0
```
Comment (Xuanwo) [inline CHANGELOG.md:14]: The first release don't need a compare.

**Review by Xuanwo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (abyss-w) [reply CHANGELOG.md:14]: Got it!

**Review by JinnyYi — APPROVED** [review]

**Pull request merged**